resources/views/layouts/app.blade.php

```diff
@@ -19,7 +19,9 @@
     <meta name="description" content="@yield('description', Rapidez::config('design/head/default_description', ''))"/>
     <meta name="robots" content="@yield('robots', Rapidez::config('design/search_engine_robots/default_robots', 'INDEX,FOLLOW'))"/>
     <link rel="canonical" href="@yield('canonical', url()->current())" />
+    <link href="{{ route('config') }}" rel="preload" as="script">
 
+    <script defer src="{{ route('config') }}"></script>
```

Comment:
`defer` is "soon enough"? We won't end up with situations where `window.config` isn't available yet?

Reply:
If we have it defined before all other scripts, and are not using it directly in the html we should be fine.

Previously we had it at the bottom of the page without defer, which should have the same effect as having it on the top of the page with defer.

Reply:
I tried my hardest to make this cause any errors and was unable to.